tests/Integration/Behaviour/Features/Context/Domain/Product/CommonProductFeatureContext.php

```diff
@@ -396,7 +396,7 @@ private function assertIndexation(string $productReference, bool $expectedIsInde
     {
         $productId = $this->getSharedStorage()->get($productReference);
         $shopId = $shopReference ? $this->getSharedStorage()->get($shopReference) : null;
-        $product = new Product($productId, false, null, $shopId);
+        $product = new Product($productId, true, null, $shopId);
```

Comment:
Why do you need a "full" product? It's not related to indexation

Reply:
I probably don't. Its a leftover from testing (when debugging indexation failures I thought it doesnt load $product->indexed field properly). Gonna revert back as it was :+1: